include/spdlog/sinks/minute_file_sink.h

```diff
@@ -0,0 +1,208 @@
+// Copyright(c) 2015-present, Gabi Melman & spdlog contributors.
+// Distributed under the MIT License (http://opensource.org/licenses/MIT)
+//File: minute_file_sink to support every 1 min file rotation.
+//Author Manoj Mallawaarachchie manoj_ws@yahoo.com / manoj@vizuamatix.com
+//example driver code:
+//auto logger = spdlog::minute_logger_mt("minutes_basic_logger", "logs/min-log.txt",false,60);
+
+#pragma once
+
+#include <spdlog/common.h>
+#include <spdlog/details/file_helper.h>
+#include <spdlog/details/null_mutex.h>
+#include <spdlog/fmt/fmt.h>
+#include <spdlog/sinks/base_sink.h>
+#include <spdlog/details/os.h>
+#include <spdlog/details/circular_q.h>
+#include <spdlog/details/synchronous_factory.h>
+
+#include <chrono>
+#include <cstdio>
+#include <ctime>
+#include <mutex>
+#include <string>
+
+namespace spdlog {
+namespace sinks {
+
+/*
+ * Generator of Minute log file names in format basename.YYYY-MM-DD-HH-MM.ext
+ */
+struct minute_filename_calculator
+{
+    // Create filename for the form basename.YYYY-MM-DD-H-M
+    static filename_t calc_filename(const filename_t &filename, const tm &now_tm)
+    {
+        filename_t basename, ext;
+        std::tie(basename, ext) = details::file_helper::split_by_extension(filename);
+        return fmt_lib::format(SPDLOG_FILENAME_T("{}_{:04d}-{:02d}-{:02d}-{:02d}_{:02d}{}"), basename, now_tm.tm_year + 1900, now_tm.tm_mon + 1,
+            now_tm.tm_mday, now_tm.tm_hour,now_tm.tm_min,ext);//colock details define in os-inl.h
+    }
+};
+
+/*
+ * Rotating file sink based on time(minutes).
+ * If truncate != false , the created file will be truncated.
+ * If max_files > 0, retain only the last max_files and delete previous.
+ */
+template<typename Mutex, typename FileNameCalc = minute_filename_calculator>
+class minute_file_sink final : public base_sink<Mutex>
```

Comment:
daily_sink and hourly_sink can change the rotation time, but `minute_file_sink` seems to have a fixed rotation time. I suggest renaming it to `every_minute_file_sink` to avoid confusion.